scipy/stats/_qmc.py

```diff
@@ -1117,14 +1117,16 @@ class Halton(QMCEngine):
     Pseudo-random number generator that generalize the Van der Corput sequence
     for multiple dimensions. The Halton sequence uses the base-two Van der
     Corput sequence for the first dimension, base-three for its second and
-    base-:math:`n` for its n-dimension.
+    base-:math:`n` for its :math:`n`-dimension, with :math:`n` the
+    :math:`n`'th prime.
 
     Parameters
     ----------
     d : int
         Dimension of the parameter space.
     scramble : bool, optional
-        If True, use Owen scrambling. Otherwise no scrambling is done.
+        If True, use random scrambling from Owen. Otherwise no scrambling
```

Comment:
```suggestion
        If True, use random scrambling from [2]_. Otherwise no scrambling
```
Is that what "Owen" refers to here?